samples/modules/thrift/hello/client/sample.yaml

```diff
@@ -0,0 +1,27 @@
+sample:
```

Comment:
all samples and tests need to depend on the module, if someone does include the module, they will not want to be building those, especially if they do not have the dependencies.

Reply:
Right - I think the module is a new symbol as of just a few days ago, so I'll add that into tests and samples thanks!

Reply:
@nashif - would this do the trick? I'll drop those in all of the `testcase.yaml` and `sample.yaml` files
```
filter: TOOLCHAIN_HAS_NEWLIB == 1 and CONFIG_ZEPHYR_THRIFT_MODULE
```

Reply:
there is a special tag twister can use to filter, look at chre for example, or nanopb

Reply:
@nashif - I think I have it now using the `modules:` tag. Is that what you mean?